### PR TITLE
Adds a bluespace vial

### PR DESCRIPTION
**Pull request by dustylens**
This PR adds a bluespace vial to the civilian research tree and corresponding fabrication recipes for the medical and science techfabs.

## About the PR

The bluespace vial holds 300 units, which seems to follow the 10x rule of bluespace containers. Given scaling this leaves it as less space efficient than the bluespace beaker but with alternative access to the centrifuge machine. So it occupies its own niche.

The production costs were meant to bridge somewhere between the bluespace syringe and beaker.

The default icon has some small alterations. Such as a metallic stopper and bluespace effects on the rim. This allows the vial to be differentiated from the standard while still retaining its fill components. I retained the standard vial components in this upload to keep everything in one place.

The vial is technically a baseitem as the standard vial has a breakable component which I felt was undesirable for a bluespace entity.

I tried my best to follow naming conventions. I recognize that some alterations may be required. The yml for the entity itself may have some baggage that I lack the skill to properly identify and remove. Though I did my best to prune it.

## Why / Balance

It felt like a niche that was missing. I feel that this will be desirable for fulfilling the role of centrifuging reagents in larger batches while serving as an attractive alternative to the blue space beaker for some reagent transport options. 

## How to test

Research vial on the civilian services track. It should be a tier 3 recipe.
Print from both the science and medical techfab.
Carry in both hands to ensure visuals are good to go.
Test fill states by filling with a reagent. Blood is ideal.
Centrifuge blood. 
Place vial in chemmaster to observe that blood has centrifuged.

Attempt to destroy the vial through throwing, explosions, harsh language. It should endure.

Throw vial. Contents should spill.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
ss+(2024-09-01+at+01 20 37)
ss+(2024-09-01+at+01 29 54)

## Breaking changes

Ideally none.

**Changelog**
:cl: dustylens
- add: Added Bluespace vial, available in protolathe and medical techfab after T3 civilian research.

### CODE REVIEW

**Comment by whatston3** [comment]
Got a few suggested changes here on [whatston3:Bluespace-Vial-suggestions](https://github.com/dustylens/frontier-station-14/compare/Bluespace-Vial...whatston3:frontier-station-14:Bluespace-Vial-suggestions?expand=1).  Nothing huge, just reparenting under BaseBeakerMetallic.

This also adds the InjectableSolution to vials as they don't currently have them, just for consistency - _seems_ reasonable enough to me, but if there's a strong reason against it, I won't push for it. (how can this fit into such a tiny opening at the top? ridiculous).  If that isn't reasonable, I think we can keep the definition as-is.  Sprites seem fine, consistent with the existing vial from what I saw.

File: Resources/Prototypes/_NF/Recipes/Lathes/chemistry.yml [hunk 1]
```diff
@@ -4,3 +4,15 @@
   completetime: 2
   materials:
     Glass: 50
+
+- type: latheRecipe
+  id: VialBluespace
+  result: VialBluespace
+  completetime: 2
+  materials:
+    Steel: 50
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Recipes/Lathes/chemistry.yml:13]:
```suggestion
    Steel: 60
```

File: Resources/Prototypes/_NF/Recipes/Lathes/chemistry.yml [hunk 2]
```diff
@@ -4,3 +4,15 @@
...
+    Plastic: 50
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Recipes/Lathes/chemistry.yml:14]:
```suggestion
    Plastic: 60
```

File: Resources/Prototypes/_NF/Recipes/Lathes/chemistry.yml [hunk 3]
```diff
@@ -4,3 +4,15 @@
...
+    Glass: 75
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Recipes/Lathes/chemistry.yml:15]:
```suggestion
```

File: Resources/Prototypes/_NF/Recipes/Lathes/chemistry.yml [hunk 4]
```diff
@@ -4,3 +4,15 @@
...
+    Plasma: 50
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Recipes/Lathes/chemistry.yml:16]:
```suggestion
    Plasma: 15
```

File: Resources/Prototypes/_NF/Recipes/Lathes/chemistry.yml [hunk 5]
```diff
@@ -4,3 +4,15 @@
...
+    Silver: 25
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Recipes/Lathes/chemistry.yml:17]:
```suggestion
    Silver: 15
```

File: Resources/Prototypes/_NF/Recipes/Lathes/chemistry.yml [hunk 6]
```diff
@@ -4,3 +4,15 @@
...
+    Bluespace: 100 #DeltaV: Bluespace
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Recipes/Lathes/chemistry.yml:18]:
```suggestion
    Bluespace: 30 #DeltaV: Bluespace
```

File: Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/lathe.yml [hunk 7]
```diff
@@ -77,8 +77,9 @@
       - JetpackVoid #Frontier
       - PowerCellHigh #Frontier
       #- PowerCellHyperNF #Frontier
-      - PowerCellMicroreactor #Frontier
+      - PowerCellMicroreactor #Frontier  
       - BluespaceBeaker
+      - VialBluespace # Frontier
```
Comment (dvir001) [inline Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/lathe.yml:82]: Should this not be in the medical techfab also?

File: Resources/Prototypes/_NF/Entities/Objects/Specific/chemistry.yml [hunk 8]
```diff
@@ -30,3 +30,40 @@
...
+    maxTransferAmount: 300
+  - type: Item
+    size: Tiny
+    sprite: _NF/Objects/Specific/Chemistry/vial_bluespace.rsi  
+  - type: StaticPrice
+    price: 100
+    vendPrice: 100  
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: null
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Entities/Objects/Specific/chemistry.yml:69]: For the price it cost it should not be able break from landing on floor just like ``BluespaceBeaker``

**Review by dvir001 — DISMISSED** [review]

**Review by whatston3 — APPROVED** [review]
Made a small set of changes: whitespace fixes, included the vial in the protolathe, and a hotfix for the plant analyzer's lathe recipe.

**Pull request merged**